Tests/TurboNavigatorTests.swift

```diff
@@ -77,6 +77,19 @@ final class TurboNavigatorTests: XCTestCase {
         XCTAssert(navigationController.viewControllers.last is VisitableViewController)
         assertVisited(url: proposal.url, on: .main)
     }
+    
+    func test_default_default_default_visitProposalResponseCancelsNavigation() {
```

Comment:
The naming was a bit strange. I feel I need more context as to why there's 3 defaults 😅.

Reply:
Ha, agreed. Each option is separated by an underscore. So in the [handled flows table](https://github.com/joemasilotti/TurboNavigator#handled-flows), this corresponds to a default _current context_, a default _given context_, and a _default given presentation_.

So I think your test should actually read `func test_none_visitProposalResponseCancelsNavigation()` since the first two options don't matter. All we care about is the third.

Let's move this to the bottom of the tests, also. To better align with the table.

Reply:
🤦🏻‍♂️ I should've RTFM. The comment at the top explains it. Thanks, Joe. Moved the test and renamed it.